### DISCUSSION TITLE
Is there any customization for image source that is not available?

### DISCUSSION THREAD

**Discussion by laichak** [igordanchenko/yet-another-react-lightbox#106] [Q&A] [answered]
Hi, thanks for creating this awesome package.

If the image source is broken or unavailable, is there any way to customize the output?

Thank you.

![image](https://user-images.githubusercontent.com/3492871/232545111-d7c44abd-9234-4454-a3bf-69ac4a30a38a.png)

**Comment by igordanchenko** [comment, marked as answer]
You can customize this icon with the `render.iconError`:

```jsx
<Lightbox
  render={{
    iconError: () => ...
  }}
  // ...
/>

**Reply by laichak** [reply]
Thank you.
